src/events/guildCreate.ts

```diff
@@ -3,8 +3,12 @@ import $ from "../core/lib";
 import {updateGlobalEmoteRegistry} from "../core/lib";
 
 export default new Event<"guildCreate">({
-    on() {
-        $.log("Updated emote registry.");
+    on(guild) {
+        $.log(
+            `[GUILD JOIN] ${guild.name} (${guild.id}) added the bot. Owner: ${guild.owner!.user.tag} (${
```

Comment:
We *might* want to make a channel configurable for this, since I'm not always watching the logs.